index.html

```diff
@@ -15,11 +15,13 @@
     <link rel="stylesheet" href="css/index.css" />
   </head>
   <body>
-    <nav class="navbar"><a href="#about">About</a></nav>
-    <nav class="navbar"><a href="#experience">Experience</a></nav>
-    <nav class="navbar"><a href="#skills">Skills</a></nav>
-    <nav class="navbar"><a href="#projects">Projects</a></nav>
-    <nav class="navbar"><a href="#connect">Connect</a></nav>
+    <nav>
+      <nav class="navbar"><a href="#about">About</a>
+      <nav class="navbar"><a href="#experience">Experience</a>
+      <nav class="navbar"><a href="#skills">Skills</a>
+      <nav class="navbar"><a href="#projects">Projects</a>
+      <nav class="navbar"><a href="#connect">Connect</a>
```

Comment:
This doesnt make sense. the `nav` that are not close. you don't even need them there

Reply:
> Hint:
```html
<nav class="navbar">
      <a href="#about">About</a>
      <a href="#experience">Experience</a>
      <a href="#skills">Skills</a>
      <a href="#projects">Projects</a>
      <a href="#connect">Connect</a>
 </nav>
 ```